src/material/paginator/paginator.ts

```diff
@@ -82,6 +82,9 @@ export class MatPaginator extends _MatPaginatorBase implements OnInit, OnDestroy
   HasInitialized {
   private _initialized: boolean;
   private _intlChanges: Subscription;
+  
+  /**to allow overwrite the enable/disable tooltip for previous/next buttons*/
+  @input() previousNextButtonsTooltipDisabled:bool;
```

Comment:
Couple of typos:

As TypeScript is case-sensitive, you have to use the capital form of `Input`.

There's also no such type as a `bool` in TypeScript: it should be `boolean`.

There should also be a whitespace in-between the variable and the type.

```suggestion
  @Input() previousNextButtonsTooltipDisabled: boolean;
```